distribution/docs/src/main/resources/_contents/_endpoints/catalog-rest-endpoint-contents.adoc

```diff
@@ -0,0 +1,72 @@
+
+==== Catalog REST Endpoint
+
+The Catalog REST Endpoint allows clients to perform CRUD operations on the Catalog using REST, a simple architectural style that performs communication using HTTP. 
+
+===== Installing the Catalog REST Endpoint
+
+The Catalog REST Endpoint is installed by default with a standard installation in the ${ddf-catalog} application.
+
+===== Configuring the Catalog REST Endpoint
+
+The RESTful CRUD Endpoint has no configurable properties.
+It can only be installed or uninstalled.
+
+===== Using the Catalog REST Endpoint
+
+The Catalog REST Endpoint provides the capability to query, create, update, and delete metacards and associated resource in the catalog provider as follows:
+
+[cols="2,1,3,4", options="header"]
+|===
+
+|Operation
+|HTTP Request
+|Details
+|Example URL
+
+|`create`
+|HTTP POST
+|HTTP request body contains the input to be ingested.
+|${public_url}/services/catalog
+
+|`update`
+|HTTP PUT
+|The ID of the Metacard to be updated is appended to the end of the URL.
+
+The updated metadata is contained in the HTTP body.
+
+|${public_url}/services/catalog/<metacardId>
```

Comment:
The `Example URL` for `update`, `delete`, `read` and `federated read` don't render properly in PDF, e.g.:

```
<a href="http://localhost:8181/services/cat alog/&lt;metacardId&gt" class="bare">http://localhost:8181/serv ices/catalog/&lt;metacardId&gt</a>; where <code>&lt;metacardId&gt;</code> is the <code>Metacard.ID</code> of the metacard to be updated
```